data/fields/electric_vehicle.json

```diff
@@ -0,0 +1,24 @@
+{
+    "keys": [
+        "bicycle",
+        "moped",
+        "motorcar",
+        "motorcycle",
+        "hgv",
+        "bus",
+        "small_electric_vehicle"
+    ],
+    "type": "manyCombo",
+    "label": "Supported vehicles",
+    "strings": {
+        "options": {
+            "moped": "Mopeds",
+            "bicycle": "E-Bikes",
```

Comment:
Consider expanding this to “Electric Bicycles”, since the UI strings tend not to use informal terminology.

Reply:
Sure, i will change this.